docs/getting_started.md

```diff
@@ -269,7 +281,17 @@ spec:
         controlPlane: "1.13.6"
 ```
 
-Run `kubectl apply -f` to apply the above files on your management cluster and it should start provisioning the new cluster.
+Run `kubectl apply -f` to apply the above files on your management cluster and it should start provisioning the new cluster:
+```bash
+$ cd out/prod-workload
+$ kubectl apply -f cluster.yaml
+cluster.cluster.k8s.io/prod-workload created
+$ kubectl apply -f machines.yaml
+machine.cluster.k8s.io/prod-workload-controlplane-1 created
+$ kubectl apply -f machineset.yaml
+machineset.cluster.k8s.io/prod-workload-machineset-1 created
+```
+
 Clusters that are provisioned by the management cluster that run your application workloads are called [Workload Clusters](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#workload-cluster).
```

Comment:
This should be at the top - a workload vs mgmt distinction needs to be made before talking about each in the doc.

Reply:
Yeah this one is tricky, there's a lot of cluster API terms here and I link to the cluster api docs at the top of the doc which has everything defined. Wondering how much of that we need to redefine here